File: Beecrowd/3-Strings/1234-Sentença_Dançante-3/1234.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        StringBuilder ans = new StringBuilder();
        boolean first = true;

        while (scanner.hasNextLine()) {
            int axu = 0;
            StringBuilder result = new StringBuilder();
            String line = scanner.nextLine();

            for (char c : line.toCharArray()) {
                if (c == ' ') {
                    result.append(" ");
                    continue;
                }
                
                if(Character.isLetter(c))axu++;
                
                result.append(axu % 2 == 1 ? Character.toUpperCase(c) : Character.toLowerCase(c));
            }

            if (!first) {
                ans.append("\n");
            }
            first = false;
            ans.append(result);
        }

        System.out.println(ans);
        scanner.close();
    }
}

File: Beecrowd/3-Strings/1234-Sentença_Dançante-3/1234.py
import sys

ans=[]
for line in sys.stdin:
    axu = 0
    result = []

    for c in line:  
        if c == ' ':
            result.append(" ")
            continue

        axu += 1
        result.append(c.upper() if axu % 2 == 1 else c.lower())

    ans.append("".join(result))
print("".join(ans), end="")
